3.7/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -70,8 +70,19 @@ zookeeper_validate() {
     check_allowed_port ZOO_PORT_NUMBER
     check_allowed_port ZOO_PROMETHEUS_METRICS_PORT_NUMBER
 
+    if is_boolean_yes "$ZOO_ADMIN_SERVER_ENABLE"; then
+      echo "Admin web server is enabled on port ${ZOO_ADMIN_SERVER_PORT_NUMBER}."
+      check_allowed_port ZOO_ADMIN_SERVER_PORT_NUMBER
+    else
+      echo "Admin web server is disabled."
+    fi
+
     check_conflicting_ports ZOO_PORT_NUMBER ZOO_PROMETHEUS_METRICS_PORT_NUMBER
 
+    if is_boolean_yes "$ZOO_ADMIN_SERVER_ENABLE"; then
+      check_conflicting_ports ZOO_PORT_NUMBER ZOO_PROMETHEUS_METRICS_PORT_NUMBER ZOO_ADMIN_SERVER_PORT_NUMBER
```

Comment:
We use 4-characters indentation, could you fix it?

```suggestion
        check_conflicting_ports ZOO_PORT_NUMBER ZOO_PROMETHEUS_METRICS_PORT_NUMBER ZOO_ADMIN_SERVER_PORT_NUMBER
```

Reply:
done